test/behaviour/connection/session/SessionSteps.java

```diff
@@ -152,4 +155,12 @@ public void sessions_in_parallel_have_databases(List<String> names) {
 
         CompletableFuture.allOf(assertions).join();
     }
+
+    @Given("set session option {word} to: {int}")
+    public void set_session_option_to(String option, int value) {
```

Comment:
We have options that are not `int`-valued. Even though we aren't testing them yet, I imagine we'll do so soon, so we should future-proof this step by renaming it to
```java
@Given("set session option {word} to int: {int}")
```

Reply:
Implemented this by always using strings and parsing them in the option setters